### PR TITLE
Include license file in distributions

### PR DESCRIPTION
**Pull request by jdufresne**
Include license file in source distribution by using `MANIFEST.in`.

Include license file in the generated wheel package. The wheel package format supports including the license file. This is done using the `[metadata]` section in the `setup.cfg` file. For additional information on this feature, see:

https://wheel.readthedocs.io/en/stable/index.html#including-the-license-in-the-generated-wheel-file

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) xlwt
```
Comment (cjw296) [inline LICENSE]: Where did you get this license from? It doesn't look correct...

**Comment by jdufresne** [comment]
In `setup.py`, the license is listed as:

```
License :: OSI Approved :: BSD License
```

The license contents were copied from:

https://choosealicense.com/licenses/bsd-3-clause/

If I misinterpreted, please let me know the correct license.

**Comment by jdufresne** [comment]
I removed the mention of xlwt from the license file. I believe that is what you were suggesting?

**Comment by cjw296** [comment]
The problem is that the license includes a copyright, and the copyright for xlwt is... not trivial.
Not sure what to do here?

**Comment by jdufresne** [comment]
Should I move https://github.com/python-excel/xlwt/blob/master/docs/licenses.rst to the root directory instead? Looks like it has already solved this problem.

**Comment by cjw296** [comment]
Yep, this approach can work well:
https://github.com/Simplistix/testfixtures/commit/814149e438da106d49fa92d3585e9c1a54a266a3

**Comment by jdufresne** [comment]
Done in latest revision.

**Pull request merged**